beamerinnerthemeusyd.sty

```diff
@@ -57,9 +58,15 @@
 \defbeamertemplate*{logo titlepage}{usyd}
 {\includegraphics[width=2.4cm]{\USYD@logo}}
 
+\defbeamertemplate*{affiliationlogo titlepage}{usyd}
+{\includegraphics[width=2.2cm]{\USYD@affiliationlogo}}
```

Comment:
We want the affiliation to be optional, so only include an image if it is defined, something like this line from the poster theme.
```
\ifdefempty{\poster@logoleft}{}{\includegraphics{\poster@logoleft}}
```